resources/views/pages/achievement/[achievement]/report-issue.blade.php

```diff
@@ -0,0 +1,148 @@
+            I met the requirements, but the achievement did not trigger.
+        </x-ticket.guide-link>
+
+        <x-ticket.guide-link text="Create Ticket"
+                             link="{{ route('achievement.create-ticket', $achievement) }}?type={{ TicketType::TriggeredAtWrongTime }}">
+            I unlocked this achievement without meeting the requirements, and then I reset it.
+        </x-ticket.guide-link>
+
+        @php
+            $subject = urlencode("Manual Unlock: $achievement->title ({$achievement->game->title})");
+            $message = urlencode("I'd like a manual unlock for [ach=$achievement->id]:\n(provide link to video/screenshot showing evidence)");
+        @endphp
+        <x-ticket.guide-link text="Request Manual Unlock"
+                             link="{{ route('message.create') }}?to=RAdmin&subject={{ $subject }}&message={{ $message }}">
+            The achievement triggered, but the unlock didn't appear on my profile.
+        </x-ticket.guide-link>
+    @else
+        <x-ticket.guide-link text="Create Ticket"
+                             link="{{ route('achievement.create-ticket', $achievement) }}?type={{ TicketType::TriggeredAtWrongTime }}">
+            I unlocked this achievement without meeting the requirements.
+        </x-ticket.guide-link>
+    @endif
+
+    @php
+        $subject = urlencode("Typo: $achievement->title ({$achievement->game->title})");
+        $message = urlencode("I'd like to report a spelling/grammar error in [ach=$achievement->id]:\n(Describe the issue here)");
+    @endphp
+    <x-ticket.guide-link text="Message QATeam"
+                         link="{{ route('message.create') }}?to=QATeam&subject={{ $subject }}&message={{ $message }}">
+        There is a spelling or grammatical error in the title or description.
+    </x-ticket.guide-link>
+
+    @php
+        $subject = urlencode("Incorrect type: $achievement->title ({$achievement->game->title})");
+        $message = urlencode("I'd like to report a misclassification error in [ach=$achievement->id]:\n(Describe the issue here)");
+    @endphp
+    <x-ticket.guide-link text="Message QATeam"
+                         link="{{ route('message.create') }}?to=QATeam&subject={{ $subject }}&message={{ $message }}">
+        The achievement type (progression/win/missable) is not correct.
+    </x-ticket.guide-link>
+
+    @php
+        $subject = urlencode("Unwelcome Concept: $achievement->title ({$achievement->game->title})");
+        $message = urlencode("I'd like to report an unwelcome concept in [ach=$achievement->id]:\n(Describe the issue here)");
+    @endphp
+    <x-ticket.guide-link text="Message DevCompliance"
+                         link="{{ route('message.create') }}?to=DevCompliance&subject={{ $subject }}&message={{ $message }}">
+        <span>The achievement contains an <a href="https://docs.retroachievements.org/Unwelcome-Concepts/">unwelcome concept</a>.</span>
```

Comment:
Recommend removing this `<span>` and wrapping the `{{ $slot }}` itself in the component with a `<p>` tag:

```php
<div class="flex items-center">
    <p>{{ $slot }}</p>
</div>
```

This does intentionally enforce that the description text must be simply text.

Reply:
done.